src/hotspot/share/gc/shenandoah/shenandoah_globals.hpp

```diff
@@ -35,6 +35,33 @@
                             range,                                          \
                             constraint)                                     \
                                                                             \
+  product(double, ShenandoahGenerationalMinOldGenGrowthPercent,             \
+          12.5, EXPERIMENTAL,                                               \
+          "(Generational mode only) If the usage within old generation "    \
+          "has grown by at least this percent of its live memory size "     \
+          "at completion of the most recent old-generation marking "        \
+          "effort, heuristics may trigger the start of a new old-gen "      \
+          "collection.")                                                    \
+          range(0.0,100.0)                                                  \
+                                                                            \
+  product(uintx, ShenandoahGenerationalIgnoreOldGrowthBelowPercentage,      \
```

Comment:
Do we want to experiment with different workloads to decide the ideal default?  10 might work even better as the ideal value for best out-of-box experience.

Reply:
Yes, 10 seems sensible. I think many people have been conditioned to hand over more than 1/2 the heap to the old generation without blinking an eye.